README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
the SVG incons in WebProfilerBundle are part of the features being delivered (as they are displayed in the UI).
However, adding an image in the repo just for the readme looks weird to me.